includes/WcfceMailProcessor.inc

```diff
@@ -416,8 +416,15 @@ class WcfceMailProcessor {
    *   An exception generated by civicrm_api3().
    */
   private function getContactIdByEmail($email) {
+    // Try to get contact id from webform first.
     $res = $this->getContactIdByEmailFromWebform($email);
 
+    // If not found - try to find a contact by primary email.
+    if (!$res) {
+      $res = $this->getContactIdByEmailFromDb($email, 1);
```

Comment:
The issue with functions like this is it's not obvious at a glance as regards what is happening and you will need to check what the second parameter is for. 

I think we should have a new function that we should use here for code readability.
```php
private function getContactIdByPrimaryEmailFromDb ($email) {
return $this->getContactIdByEmailFromDb($email, 'primary')
}
```

Reply:
If we change argument to a string, I think it's fine to have it like this:
```php
$this->getContactIdByEmailFromDb($email, 'primary')
```

Do we really  need dozens of similar functions?

Reply:
Okay, fair enough.